src/components/radio-group/radio-group.ts

```diff
@@ -39,6 +87,63 @@ export default class SlRadioGroup extends LitElement {
     });
   }
 
+  getAllRadios(): SlRadio[] {
+    return [...this.querySelectorAll('sl-radio')];
+  }
+
+  checkRadioByIndex(index: number): SlRadio[] {
+    const radios = this.deselectAll();
+
+    radios[index].focus();
+    radios[index].checked = true;
+    this._value = radios[index].value;
+
+    return radios;
+  }
+
+  deselectAll(): SlRadio[] {
```

Comment:
Okay, first copy/pasta was reasonable, but do these methods make sense to expose and document?

Reply:
Only public API methods should have JSDoc comments. The rest are pseudo-private methods that don’t get published. Is this intended to be public?